pixeltable/utils/arrow.py

```diff
@@ -0,0 +1,112 @@
+import pyarrow as pa
+from typing import Any, Dict, Iterator, Iterable, Optional, Union, List
+import pixeltable.type_system as ts
+import pathlib
+import logging
+
+_logger = logging.getLogger(__name__)
+
+_pa_to_pt : Dict[pa.DataType, ts.ColumnType] = {
+        pa.string() : ts.StringType(nullable=True),
+        pa.timestamp('us') : ts.TimestampType(nullable=True),
+        pa.bool_() : ts.BoolType(nullable=True),
+        pa.uint8() : ts.IntType(nullable=True),
+        pa.int8() : ts.IntType(nullable=True),
+        pa.uint32() : ts.IntType(nullable=True),
+        pa.uint64() : ts.IntType(nullable=True),
+        pa.int32() : ts.IntType(nullable=True),
+        pa.int64() : ts.IntType(nullable=True),
+        pa.float32() : ts.FloatType(nullable=True),
+}
+
+_pt_to_pa : Dict[type, pa.DataType] = {
+        ts.StringType : pa.string(),
+        ts.TimestampType : pa.timestamp('us'), # postgres timestamp is microseconds
+        ts.BoolType : pa.bool_(),
+        ts.IntType : pa.int64(),
+        ts.FloatType : pa.float32(),
+        ts.JsonType : pa.string(), # TODO(orm) pa.struct() is possible
+        ts.ImageType : pa.binary(), # inline image
+        ts.AudioType : pa.string(), # path
+        ts.VideoType : pa.string(), # path
+        ts.DocumentType : pa.string(), # path
+}
+
+def get_pixeltable_type(arrow_type : pa.DataType) -> ts.ColumnType:
```

Comment:
I prefer `to_pixeltable_type` etc but don't feel super strongly about it.

Reply:
will do